test/parallel/test-whatwg-url-origin.js

```diff
@@ -50,3 +51,12 @@ function runURLTests(urltests) {
 
 runURLOriginTests()
 /* eslint-enable */
+
+// Tests below are not from WPT.
+const tests = require(path.join(common.fixturesDir, 'url-tests-additional.js'));
```

Comment:
I think it's good to keep it since potentially a case can be added into `url-tests-addition` and some test codes actually are still having it.
e.g.
+ https://github.com/nodejs/node/blob/master/test/parallel//test-whatwg-url-parsing.js#L27-L43
+ https://github.com/nodejs/node/blob/master/test/parallel/test-whatwg-url-inspect.js#L17-L18